net/ntopng/src/opnsense/service/templates/OPNsense/Ntopng/ntopng.conf

```diff
@@ -12,4 +12,5 @@
 {%   if helpers.exists('OPNsense.ntopng.general.dnsmode') and OPNsense.ntopng.general.dnsmode != '' %}
 -n={{ OPNsense.ntopng.general.dnsmode }}
 {%   endif %}
+-d=/var/db/ntopng
```

Comment:
Shouldn't this be made persistent under /var MFS? We don't seem to do this yet.

Reply:
Reference code please. :) Never worked with mfs

Reply:
https://github.com/opnsense/plugins/blob/master/net/zerotier/src/opnsense/service/templates/OPNsense/zerotier/zerotier#L9

Reply:
Might be a good time to introduce this as the directory changes anyway. Also another bug fixed for the changelog :)

Reply:
I have no idea why it's outside of if/else but you might know best :)